grafana-plugin/src/components/Policy/EscalationPolicy.tsx

```diff
@@ -381,39 +377,8 @@ export class EscalationPolicy extends React.Component<EscalationPolicyProps, any
     );
   }
 
-  private _renderTriggerCustomAction() {
```

Comment:
wouldn't need this be for backwards compatibility? not entirely familiar with this part of code, but looks like needed. or were the old webhooks migrated?

Reply:
> or were the old webhooks migrated?

bingo! on the backend we migrated them to "new webhooks". They'd use this `custom_webhook` branch [above on lines 112-113](https://github.com/grafana/oncall/pull/2536/files/76eef1aa5a9738bfb1d4fab97a2d8398aaa4e8cc#diff-83b0920a6b120aeab5bb1260e601f56b849fa8000ffa2d99980f7641f6f68b57L114-R113)